2020-09-19 at 20:16:00 | DEBUG | Checking if feeds have updated...
2020-09-19 at 20:16:00 | DEBUG | Checking if feed: hacker_news requires updating...
2020-09-19 at 20:16:01 | DEBUG | Feed: hacker_news does not require any updates.
2020-09-19 at 20:16:01 | DEBUG | Checking if feed: mac_rumors requires updating...
2020-09-19 at 20:16:01 | DEBUG | Feed: mac_rumors does not require any updates.
2020-09-19 at 20:16:01 | DEBUG | Checking if feed: zhihu_daily requires updating...
2020-09-19 at 20:16:02 | DEBUG | Feed: zhihu_daily does not require any updates.
2020-09-19 at 20:16:02 | DEBUG | Checking if feed: NASA requires updating...
2020-09-19 at 20:16:02 | DEBUG | Feed: NASA does not require any updates.
2020-09-19 at 20:16:02 | DEBUG | Checking if feed: National Geography requires updating...
2020-09-19 at 20:16:03 | DEBUG | Feed: National Geography does not require any updates.
2020-09-19 at 20:16:03 | DEBUG | Sleeping for 12 hours...
2020-09-20 at 07:59:32 | DEBUG | Update: {'update_id': 753447604, 'message': {'message_id': 101, 'date': 1600559949, 'chat': {'id': 710577478, 'type': 'private', 'username': 'zephyrfun', 'first_name': '哇', 'last_name': '刮刮乐'}, 'text': '/weather', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 8}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 710577478, 'first_name': '哇', 'is_bot': False, 'last_name': '刮刮乐', 'username': 'zephyrfun', 'language_code': 'en'}}} caused the error: list index out of range
2020-09-20 at 08:15:58 | DEBUG | Checking if feeds have updated...
2020-09-20 at 08:15:58 | DEBUG | Checking if feed: hacker_news requires updating...
2020-09-20 at 08:15:59 | DEBUG | Feed: hacker_news requires updating! Running date_title for feeds.json.
2020-09-20 at 08:16:00 | DEBUG | Checking if feed: mac_rumors requires updating...
2020-09-20 at 08:16:01 | DEBUG | Feed: mac_rumors requires updating! Running date_title for feeds.json.
2020-09-20 at 08:16:01 | DEBUG | Checking if feed: zhihu_daily requires updating...
2020-09-20 at 08:16:02 | DEBUG | Feed: zhihu_daily requires updating! Running date_title for feeds.json.
2020-09-20 at 08:16:03 | DEBUG | Checking if feed: NASA requires updating...
2020-09-20 at 08:16:03 | DEBUG | Feed: NASA does not require any updates.
2020-09-20 at 08:16:03 | DEBUG | Checking if feed: National Geography requires updating...
2020-09-20 at 08:16:04 | DEBUG | Feed: National Geography does not require any updates.
2020-09-20 at 08:16:04 | DEBUG | Sleeping for 12 hours...
2020-09-20 at 15:36:07 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None)))
2020-09-20 at 15:36:08 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:09 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:12 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:18 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:29 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:40 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:36:51 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 15:37:20 | DEBUG | Update: None caused the error: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot1287844941:AAGEqYC6ahcMfv_z2r6n4tAQsg0i3drM5VE/getUpdates (Caused by ProxyError('Cannot connect to proxy.', ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None)))
2020-09-20 at 20:15:58 | DEBUG | Checking if feeds have updated...
2020-09-20 at 20:15:58 | DEBUG | Checking if feed: hacker_news requires updating...
2020-09-20 at 20:15:59 | DEBUG | Feed: hacker_news does not require any updates.
2020-09-20 at 20:15:59 | DEBUG | Checking if feed: mac_rumors requires updating...
2020-09-20 at 20:16:01 | DEBUG | Feed: mac_rumors does not require any updates.
2020-09-20 at 20:16:01 | DEBUG | Checking if feed: zhihu_daily requires updating...
2020-09-20 at 20:16:02 | DEBUG | Feed: zhihu_daily does not require any updates.
2020-09-20 at 20:16:02 | DEBUG | Checking if feed: NASA requires updating...
2020-09-20 at 20:16:08 | DEBUG | Feed: NASA requires updating! Running date_title for feeds.json.
2020-09-20 at 20:16:09 | DEBUG | Checking if feed: National Geography requires updating...
2020-09-20 at 20:16:11 | DEBUG | Feed: National Geography requires updating! Running date_title for feeds.json.
2020-09-20 at 20:16:11 | DEBUG | Sleeping for 12 hours...
2020-09-21 at 08:15:58 | DEBUG | Checking if feeds have updated...
2020-09-21 at 08:15:58 | DEBUG | Checking if feed: hacker_news requires updating...
2020-09-21 at 08:15:59 | DEBUG | Feed: hacker_news does not require any updates.
2020-09-21 at 08:15:59 | DEBUG | Checking if feed: mac_rumors requires updating...
2020-09-21 at 08:16:00 | DEBUG | Feed: mac_rumors requires updating! Running date_title for feeds.json.
2020-09-21 at 08:16:01 | DEBUG | Checking if feed: zhihu_daily requires updating...
2020-09-21 at 08:16:01 | DEBUG | Feed: zhihu_daily requires updating! Running date_title for feeds.json.
2020-09-21 at 08:16:02 | DEBUG | Checking if feed: NASA requires updating...
2020-09-21 at 08:16:03 | DEBUG | Feed: NASA does not require any updates.
2020-09-21 at 08:16:03 | DEBUG | Checking if feed: National Geography requires updating...
2020-09-21 at 08:16:03 | DEBUG | Feed: National Geography does not require any updates.
2020-09-21 at 08:16:03 | DEBUG | Sleeping for 12 hours...
